packages/zoe/src/zoe.js

```diff
@@ -240,111 +240,113 @@ function makeZoe(vatAdminSvc) {
         details`${installationHandle} was not a valid installationHandle`,
       );
       const publicApiP = makePromiseKit();
-      return E(vatAdminSvc)
-        .createVat(zcfContractBundle)
-        .then(({ root, adminNode }) => {
-          /** @type {{ startContract: StartContract }} */
-          const zcfRoot = root;
-          const instanceHandle = makeHandle('InstanceHandle');
-          const zoeForZcf = makeZoeForZcf(instanceHandle, publicApiP);
-
-          const cleanedKeywords = cleanKeywords(issuerKeywordRecord);
-          const issuersP = cleanedKeywords.map(
-            keyword => issuerKeywordRecord[keyword],
-          );
-          const makeCleanup = _marker => {
-            return () => {
-              // console.log(`ZOE makeInstance  enter CLEANUP: ${marker} `);
-              const { offerHandles } = instanceTable.get(instanceHandle);
-              // This cleanup can't rely on ZCF to complete the offers since
-              // it's invoked when ZCF is no longer accessible.
-              completeOffers(instanceHandle, Array.from(offerHandles));
-            };
+
+      const vatP = zcfBundleName
+        ? E(vatAdminSvc).createVatByName(zcfBundleName)
+        : E(vatAdminSvc).createVat(zcfContractBundle);
```

Comment:
Hmm, is there any reason to keep both ways?

Reply:
This was me being conservative -- so things will still work the old way if whatever code is setting up to run a swingset with Zoe in it doesn't have a way to pre-position the bundle.  This may apply, for example, to some of the test setups.